tenders/hvt/hvt_openbsd_x86_64.c

```diff
@@ -190,6 +190,14 @@ int hvt_vcpu_loop(struct hvt *hvt)
                     hvt_gpa_t gpa = vei->vei.vei_data;
                     fn(hvt, gpa);
                     break;
+#if defined(VMM_IOC_MPROTECT_EPT)
```

Comment:
Minor: Is `VMX_EXIT_EPT_VIOLATION` dependent on the presence of `VMM_IOC_MPROTECT_EPT`? In other words, would this case be useful, and can it compile on, OpenBSD < 6.7? If yes then there's no need for the `#ifdef` here.

Reply:
@mato kinda yes, they were both added in the same commit, if won't compile on OpenBSD < 6.7